os_migrate/playbooks/deploy_conversion_hosts.yml

```diff
@@ -20,6 +20,7 @@
         os_migrate_conversion_net_mtu: "{{ os_migrate_src_conversion_net_mtu|default(omit) }}"
         os_migrate_conversion_subnet_dns_nameservers:
           "{{ os_migrate_src_conversion_subnet_dns_nameservers|default(omit) }}"
+        os_migrate_conversion_fixed_floating_ip_address: "{{ os_migrate_src_conversion_floating_ip_address|default(false) }}"
```

Comment:
Should we fix the inner name too, to keep consistency in the variable naming?

Reply:
yeah lets have it all the same, done

Reply:
Thanks :)